numba/core/ir_utils.py

```diff
@@ -819,20 +852,31 @@ def find_potential_aliases(blocks, args, typemap, func_ir, alias_map=None,
                 # only mutable types can alias
                 if is_immutable_type(lhs, typemap):
                     continue
-                if isinstance(expr, ir.Var) and lhs!=expr.name:
+                if isinstance(expr, ir.Var) and lhs != expr.name:
                     _add_alias(lhs, expr.name, alias_map, arg_aliases)
                 # subarrays like A = B[0] for 2D B
-                if (isinstance(expr, ir.Expr) and (expr.op == 'cast' or
-                    expr.op in ['getitem', 'static_getitem'])):
+                if (
+                    isinstance(expr, ir.Expr)
+                    and (
+                        expr.op == 'cast'
+                        or expr.op in ['getitem', 'static_getitem']
+                    )
+                ):
                     _add_alias(lhs, expr.value.name, alias_map, arg_aliases)
                 # array attributes like A.T
-                if (isinstance(expr, ir.Expr) and expr.op == 'getattr'
-                        and expr.attr in ['T', 'ctypes', 'flat']):
+                if (
+                    isinstance(expr, ir.Expr)
+                    and expr.op == 'getattr'
+                    and expr.attr in ['T', 'ctypes', 'flat']
+                ):
                     _add_alias(lhs, expr.value.name, alias_map, arg_aliases)
                 # a = b.c.  a should alias b
-                if (isinstance(expr, ir.Expr) and expr.op == 'getattr'
-                        and expr.attr not in ['shape']
-                        and expr.value.name in arg_aliases):
+                if (
+                    isinstance(expr, ir.Expr)
+                    and expr.op == 'getattr'
+                    and expr.attr not in ['shape']
+                    and expr.value.name in arg_aliases
+                ):
```

Comment:
Does this help with `flake8`? Is this coming from an auto-formatter? I'm not sure it's actually making it more readable.

Reply:
It's not from an auto-formatter, but it is inspired by the format of the autoformatter Black - see [above comment](https://github.com/numba/numba/pull/7195#discussion_r706624366). IMO it's considerably more readable to layout conditions like this, but I get the impression you don't agree - I can revert these if you'd prefer.